purecloud_api_dotnet_oauth_control.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
     <metadata>
         <id>PureCloudOAuthControl</id>
-        <version>2.1.0.0</version>
+        <version>1.3.0.0</version>
```

Comment:
I'm guessing this should be a higher value

Reply:
yes i just realised that. i was looking at the release version on the repo when i made that change.